lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,10 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
+      run!(cmd('hostnamectl'), :params => ['set-hostname', name])
```

Comment:
oh noes. That is a bug. [single letter](https://github.com/ManageIQ/awesome_spawn/commit/2a03df8aa0567a63332512bfb7fa427ddc003d0e)